<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateticket</class>
 <widget class="QMainWindow" name="updateticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UPDATE DETAILS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/SAAD COMMUNICATION/Downloads/chrome_hqmSI9koUf (2).png</normaloff>C:/Users/SAAD COMMUNICATION/Downloads/chrome_hqmSI9koUf (2).png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>340</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 1px solid #ffffff ;
border-radius: 7px;
background-color: rgb(94, 51, 134);</string>
    </property>
    <property name="text">
     <string>Update Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
border: 1px solid #ffffff ;
border-radius: 7px;
background-color: rgb(94, 51, 134);</string>
    </property>
    <property name="text">
     <string>Back </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>691</width>
      <height>371</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/SAAD COMMUNICATION/Downloads/how-to-update-wordpress.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-40</y>
      <width>1341</width>
      <height>711</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(20, 19, 120);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>681</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Castellar</family>
      <pointsize>25</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Updating Ticket Details</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Footlight MT Light</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TicketID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/SAAD COMMUNICATION/Downloads/Untitled (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>242</y>
      <width>341</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Palatino Linotype</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(19, 18, 113);
border: 2px solid #FFFFFF;
border-radius: 7px;</string>
    </property>
    <property name="placeholderText">
     <string>Enter the TicketID</string>
    </property>
   </widget>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>label</zorder>
   <zorder>label_4</zorder>
   <zorder>lineEdit</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
